base_maintenance/views/report_maintenance_request.xml

```diff
@@ -106,14 +106,19 @@
                 </t>
             </t>
         </template>
-        <report
-            id="action_report_maintenance_request"
-            string="Maintenance Request"
-            model="maintenance.request"
-            report_type="qweb-pdf"
-            file="base_maintenance.report_maintenance_request"
-            name="base_maintenance.report_maintenance_request"
-            print_report_name="'Maintenance Request - %s' % (object.name)"
-        />
+        <record id="action_report_maintenance_request" model="ir.actions.report">
```

Comment:
The report doesnt appear in the request's form

Reply:
@jarroyomorales  The problem is that report record needs binding_type and binding_model_id fields. The first one is alreaady fixed, but the second one not yet. On runbot I've clicked on "Add in the print menu" button to achieve it.

In my local environment adding this line in order to permanently fix it:

`<field name="binding_model_id" ref="model_maintenance_request" />`

causes a parse error. And I didn't solve it yet... :(

Reply:
Oh, I tested on runbot and saw the button, thats why I approved the PR. 
Maybe `ref="maintenance.model_maintenance_request"` solves it?

Reply:
Thank you so much! :D